Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/ShiftController.cs

```diff
@@ -165,8 +164,15 @@ public async Task<App.KronosWfc.Models.ResponseEntities.Shifts.UpcomingShifts.Re
         /// <param name="user">The user the shift is for.</param>
         /// <param name="mappedTeam">The team the user is in.</param>
         /// <returns>A response for teams.</returns>
-        public async Task<ShiftsIntegResponse> DeleteShiftFromKronos(ShiftsShift shift, AllUserMappingEntity user, TeamToDepartmentJobMappingEntity mappedTeam)
+        public async Task<ShiftsIntegResponse> DeleteShiftFromKTeamsAsync(ShiftsShift shift, AllUserMappingEntity user, TeamToDepartmentJobMappingEntity mappedTeam)

```

Comment:
The purpose of this method is to propagate the delete made in Teams to Kronos therefore I think the original name DeleteShiftFromKronos or maybe DeleteShiftInKronos makes more sense?